client/src/pages/board/components/Task.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {Card, Typography} from '@material-ui/core';
+import {Draggable} from 'react-beautiful-dnd';
+import {makeStyles} from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  drag: {
+    width: 275,
+    minHeight: '20vh',
+    padding: theme.spacing(1),
+    borderRadius: 4,
+    backgroundColor: '#F4F6FF',
+    border: '1px solid red',
+  },
+  card: {
+    userSelect: 'none',
+    padding: theme.spacing(2),
+    margin: theme.spacing(1),
+  },
+  'card-footer': {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+}));
+
+export default function Task({task, index}) {
+  const classes = useStyles();
+
+  return (
+    <Draggable draggableId={task.id} index={index}>
+      {(provided) => (
```

Comment:
It's a good start at tidying up our codebase for the longterm. The beautiful-dnd convention is easier to follow alongside documentation for similar reasons I would backlog refactoring onDragEnd or the 'provided' and 'snapshot' objects in Column and KanbanBoard jsx's. This will be easy to do for the Task component because it doesn't have any nested draggables/droppables so I'll implement your suggested change. Aww yee.